Dockerfile-dev

```diff
@@ -13,6 +13,7 @@ COPY requirements.txt .
 COPY metadata_backend/ ./metadata_backend
 
 RUN pip install .
+RUN pip install authlib==0.15.5  # required for mockauth (integration test)
```

Comment:
i would not fix the version, as it would be difficult to keep up to date

Reply:
removed version

Reply:
I initially tried having authlib in `requirements-dev`, but it didn't help. I found out, `authlib` must be installed in the docker image before for this line to run https://github.com/CSCfi/metadata-submitter/blob/develop/docker-compose.yml#L63

I'll remove the version